packages/common/src/common.ts

```diff
@@ -1086,6 +1088,20 @@ export class Common {
     return copy
   }
 
+  /**
+   * Initialize the KZG library
+   * @param kzg - a KZG implementation that implements the @ethereumjs/util KZG interface
+   * @param pathToTrustedSetup string representing the path to the trusted setup necessary for
+   * instantiating the KZG object - defaults to the official one shipped with EthereumJS
+   */
+  initializeKZG(kzg: Kzg, pathToTrustedSetup?: string) {
+    try {
+      initKZG(kzg, pathToTrustedSetup ?? __dirname + '/../../client/src/trustedSetups/official.txt')
```

Comment:
This is for sure not working just realizing. 🙂 😂

Reply:
Hah, true.  Going to revert this one.